platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/KeysAndCerts.java

```diff
@@ -171,6 +179,11 @@ public static KeysAndCerts generate(
         agrDetRandom.setSeed(AGR_SEED);
         agrKeyGen.initialize(CryptoConstants.AGR_KEY_SIZE_BITS, agrDetRandom);
 
+        tssEncryptionKeyRandom.setSeed(masterKey);
```

Comment:
Repeated code for setting seeds for different key generators. could be refactored/extracted into separate methods.